Formula/llnode.rb

```diff
@@ -20,7 +20,11 @@ class Llnode < Formula
   end
 
   resource "lldb" do
-    if MacOS::Xcode.version >= "8.0"
+    if MacOS::Xcode.version >= "8.3"
```

Comment:
Note: required lldb version changes with xcode updates, trying to keep the recent one here.